src/legacy/ManifestTranslator.ts

```diff
@@ -182,7 +179,17 @@ export class ManifestTranslator {
     app.store.type = storeType;
 
     app.extensions = [];
-    app.api = firstExt.api;
+    if (firstExt.api) {
+      app.api = new ManifestApi();
+      app.api.client = new ManifestApiClient();
+      app.api.scopes = firstExt.api.scopes;
+      app.api.client.id = firstExt.api.applicationId;
+      app.api.redirectUris = [firstExt.api.redirectUri];
```

Comment:
Shouldn't this take into account also `redirecUris` ?
I.e.
```suggestion
      app.api.redirectUris = firstExt.api.redirectUris ?? [firstExt.api.redirectUri];
```

Reply:
It is a mapping of V1 manifests to V2 manifests. There is no redirectUris in V1.